### ISSUE TITLE
Chocolatey package for cli install on Windows.

### ISSUE THREAD

**Issue by amanbha** [dapr/cli#172]
## Describe the proposal
Provide chocolatey package for install on Windows.

**Comment by christophwille** [comment]
Yes, please, much like https://chocolatey.org/packages/azure-functions-core-tools. A `choco update all -Y` makes live so much easier.

**Comment by masters3d** [comment]
+1 pls

**Comment by NileshGule** [comment]
Agree with requesters that managing packages with Chocolatey is much easier as compared to running Powershell script.

**Comment by shubham1172** [comment]
Now that we have winget, do we still need this? Winget comes with the new Windows out of box (with App installer), so do we really need to support chocolatey now? I vote to close this issue in favor of #424

**Comment by christophwille** [comment]
I for one built my dev machine installation around chocolatey long before winget was a thing (see https://gist.github.com/christophwille/87a88762e2fd79cf52e59deab7af93ff).
